Cpix/DrmSystemCollection.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Axinom.Cpix.Internal;
+using System;
+using System.Linq;
+using System.Xml;
+
+namespace Axinom.Cpix
+{
+	sealed class DrmSystemCollection : EntityCollection<DrmSystem>
+	{
+		public const string ContainerXmlElementName = "DRMSystemList";
+
+		public DrmSystemCollection(CpixDocument document) : base(document)
+		{
+		}
+
+		internal override string ContainerName => ContainerXmlElementName;
+
+		protected override XmlElement SerializeEntity(XmlDocument document, XmlNamespaceManager namespaces, XmlElement container, DrmSystem entity)
+		{
+			var drmSystemElement = new DrmSystemElement
+			{
+				SystemId = entity.Id,
+				KeyId = entity.KeyId,
+				ContentProtectionData = entity.ContentProtectionData
+			};
+
+			return XmlHelpers.AppendChildAndReuseNamespaces(drmSystemElement, container);
```

Comment:
You asked how ContentProtectionData can be merged into DASH manifest while taking proper care of namespaces. I guess this would be the way to do it!

Reply:
Thanks.